p2p/transport/webrtc/pb/message.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+
+package webrtc.pb;
+
+message Message {
+  enum Flag {
+    // The sender will no longer send messages on the stream.
+    FIN = 0;
+    // The sender will no longer read messages on the stream. Incoming data is
+    // being discarded on receipt.
+    STOP_SENDING = 1;
```

Comment:
If you rename `FIN` to `STOP_SENDING` and `STOP_SENDING` to `STOP_PROCESSING`, the naming mights be a better aligned with these comments?

Reply:
Naming originates from https://github.com/libp2p/specs/blob/master/webrtc/README.md#multiplexing. I suggest keeping those two consistent.

Reply:
Fair enough, it's a shame, but reasonable indeed. In that case perhaps add that as a remark in the comment that we have "bad names" on purpose and with good reason (see reasons).